backend/dataall/base/aws/parameter_store.py

```diff
@@ -29,37 +29,29 @@ def client(AwsAccountId=None, region=None, role=None):
     def get_parameter_value(AwsAccountId=None, region=None, parameter_path=None):
         if not parameter_path:
             raise Exception('Parameter name is None')
-        try:
-            parameter_value = ParameterStoreManager.client(AwsAccountId, region).get_parameter(Name=parameter_path)[
-                'Parameter'
-            ]['Value']
-        except ClientError as e:
-            raise Exception(e)
+        parameter_value = ParameterStoreManager.client(AwsAccountId, region).get_parameter(Name=parameter_path)[
+            'Parameter'
+        ]['Value']
```

Comment:
what are the implications of removing the try-except for other parts of the code?

Reply:
I see that it is used by the pivot_role utils, maintenance_service, other API calls

Reply:
@dlpzx these try-except blocks make no sense. Even more, they make obstacles for error handling.
1. All of them simply catch error and do nothing but throw it further
2. Instead of ClientError (which inherits Exception) there blocks just throw basic Exception making it hard to distinguish different types of errors.
We had conversation with @petrkalos and decided to drop these statements